packages/server/modules/core/repositories/models.ts

```diff
@@ -0,0 +1,14 @@
+import { StoreModel } from '@/modules/core/domain/projects/operations'
+import { createBranchFactory } from '@/modules/core/repositories/branches'
+import { Knex } from 'knex'
+
+export const storeModelFactory =
```

Comment:
While on a general level i do agree with you, here, its not about this function here, but the nomenclature of a new service functions, the `createNewProjectFactory`, where i really do not want the old terminology to contaminate things.

Reply:
@gjedlicska The same reasoning can be applied for pretty much all new code, since all new code relates to "projects/models/versions" not "streams/branches/commits", and then by following it we'll just end up with a ton of these wrappers for everything. I would just call the key of the dep `storeModel` in the service function's dep object, but requrie the type to be the original/pre-existing `StoreBranch` or whatever its called.

```
(deps: {
  storeModel: StoreBranch
}) => ...
```

Reply:
Doing a type alias  is not what I need, I need to not have the `streamId` argument in the input.

And yes, we'll have to make due with some translation layers as we move toward a new terminology only codebase. But we have to do that translation gradually.

Reply:
> we'll have to make due with some translation layers
Why tho? The downsides outweigh the only minor benefit of having "nicer names". We can always rename the entire function and its args now that everything is in TS, if its that important.

Since this is the only remaining issue in the PR feel free to merge it, but I disagree with this strategy